SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -0,0 +1,667 @@
+#
+# spec file for package libcontainers-common
+#
+# Copyright (c) 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+# commonver - version from containers/common
+%define commonver 0.14.6
+# podman - version from containers/podman
+%define podmanver 2.0.3
+# storagever - version from containers/storage
+%define storagever 1.20.2
+# imagever - version from containers/image
+%define imagever 5.5.1
+Summary:        Configuration files common to github.com/containers
+Name:           libcontainers-common
+Version:        20200727
+Release:        2%{?dist}
+License:        Apache-2.0 AND GPL-3.0+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Management
+URL:            https://github.com/containers
+#Source0:       https://github.com/containers/image/archive/v5.5.1.tar.gz
+Source0:        image-%{imagever}.tar.gz
```

Comment:
Please re-name `Source0`, `Source1`, `Source7`, and `Source9` to contain the name of the package. Otherwise, potential file name conflicts are increasingly more possible as we move forward.